wp-content/themes/goonj-crm/main.js

```
@@ -179,3 +179,78 @@ document.addEventListener("DOMContentLoaded", function() {
+
+        var selectedValue = selectedRadio.value;
+        console.log('Selected value:', selectedValue);
+
+        if (selectedValue === '2') { // "Donate Monthly" is selected
+            console.log('Showing recurring section');
+            recurringSection.style.setProperty('display', 'block', 'important');
+            if (isRecurCheckbox) {
+                isRecurCheckbox.checked = true;
+                console.log('is_recur checkbox checked (Donate Monthly selected)');
+                // Dispatch change event to ensure CiviCRM updates the form
+                var changeEvent = new Event('change');
```

Biome (1.9.4)

[error] Shouldn't redeclare 'changeEvent'. Consider to delete it or rename it.

'changeEvent' is defined here:


(lint/suspicious/noRedeclare)